packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -154,7 +154,8 @@ describeNoCompat("Message size", (getTestObjectProvider) => {
 		assertMapValues(remoteMap, messageCount, largeString);
 	});
 
-	itExpects(
+	// Blocked waiting on AB#2690
+	itExpects.skip(
```

Comment:
It would be great to see this test changed to reflect the new batch layout to get some coverage for it. It should no longer fail with your changes, as batches will be submitted separately. Can you exclude only the local server?